### DISCUSSION TITLE
Changing burner area

### DISCUSSION THREAD

**Discussion by FireStudent321** [firemodels/cfast#2124] [General]
Hi,

I was thinking of changing just the area of the burner and keeping the other parameters the same. I was just wondering if changing the area of the burner only would change my temperature results even though the same HRR are achieved in FDS. Will this affect the measurements at the other side of the room for instance?

Thanks.

**Comment by drjfloyd** [comment]
You could review the CFAST Technical Reference and study the equations for how CFAST determines how much air to entrain into the upper layer (which determines the layer temperature) and think if those equations would change if you changed the pool diameter.

You could also run two simple test cases where everything is the same except for the burner area and look at the results.
